pkg/apimachinery/registered/registered.go

```diff
@@ -99,12 +121,6 @@ func IsEnabledVersion(v unversioned.GroupVersion) bool {
 	return found
 }
 
-// IsRegisteredVersion returns if a version is registered.
-func IsRegisteredVersion(v unversioned.GroupVersion) bool {
```

Comment:
I remember @deads2k specifically asked me to add these functions when reviewing my PR. @deads2k do you still want to keep these?